### DISCUSSION TITLE
How do i build a PHPMailer

### DISCUSSION THREAD

**Discussion by larryscottmd** [PHPMailer/PHPMailer#2353] [General]
Good day folks,

Please am a newbie and would like to learn how to build a PHPMailer....

**Comment by aleksandrmelnikov** [comment]
Can you give more details on what you're trying to accomplish?
Are you trying to implement phpmailer functionality
From scratch?
Or are you trying to learn what you need to solve to recreate phpmailer?

**Reply by larryscottmd** [reply]
@aleksandrmelnikov, yes...i want to learn how to implement phpmailer functionality from the scratch...

**Reply by Synchro** [reply]
@aleksandrmelnikov tread carefully – "a phpmailer" is often used as a descriptive term for a spambot, rather than a reference to this package. I get these kind of requests quite regularly.

@larryscottmd Can you describe exactly what you're trying to achieve?

**Reply by larryscottmd** [reply]
@Synchro, for email marketing purpose for my product and what's spambot....?

**Reply by Synchro** [reply]
I'd really recommend using a complete system for doing that, like PHPList or Mailcoach. Building your own is difficult. The mailing list example provided with PHPMailer will allow you to send a message to addresses you pull from a database, but teaching you how to build a complete system is out of scope for here.

**Reply by larryscottmd** [reply]
@Synchro, can you suggest or recommend where to get a complete PHPmailer system?

**Reply by Synchro** [reply]
I recommended PHPList and Mailcoach in my previous message.

**Comment by aleksandrmelnikov** [comment]
Thank you for the heads up.
I definitely don't want to spend my time contributing to spam =\.

**Reply by 8912** [reply]
噗，笑死我了哈哈哈
